docs/modules/ROOT/partials/setup_terraform.adoc

```diff
@@ -1,37 +1,36 @@
 . Setup Terraform
 +
-Prepare terraform
-+
-[source,console]
+.Prepare Terraform execution environment
+[source,bash]
 ----
 # Set terraform image and tag to be used
 tf_image=$(\
-  yq eval ".parameters.openshift4_cloudscale.images.terraform.image" \
-  dependencies/openshift4-cloudscale/class/defaults.yml)
+  yq eval ".parameters.openshift4_terraform.images.terraform.image" \
+  dependencies/openshift4-terraform/class/defaults.yml)
 tf_tag=$(\
-  yq eval ".parameters.openshift4_cloudscale.images.terraform.tag" \
-  dependencies/openshift4-cloudscale/class/defaults.yml)
+  yq eval ".parameters.openshift4_terraform.images.terraform.tag" \
+  dependencies/openshift4-terraform/class/defaults.yml)
 
+pushd catalog/manifests/openshift4-terraform/
```

Comment:
Move this to the end of this code block, and adjust the creation of the `terraform.env` files to be

```
cat <<EOF > ./terraform.env
...
EOF
```

Alternatively, remove the whole `base_dir` construction, and just use `--env-file ./terraform.env` in the alas.